src/js/map/popup.js

```diff
@@ -1,5 +1,8 @@
 import {Observable, setPopupStyle, formatNumericalValue} from '../utils';
+import {format as d3format} from "d3-format/src/defaultLocale";
 
+const PERCENTAGE_TYPE = 'Percentage';
```

Comment:
I think we should move those into a `constant.js` and export them and use them here as well:
https://github.com/OpenUpSA/wazimap-ng-ui/blob/staging/src/js/profile/chart.js#L10